.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+on:
+  pull_request:
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.18
+      uses: actions/setup-go@v3
+      with:
+        go-version: ^1.18
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v3  
+    - name: Build
+      run: |
+       make gomod
```

Comment:
I don't think we need to do `gomod`. That should have been done by the user before.